src/components/Common/AutoFocusInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <b-form-input
+    :ref="refName"
+    v-model="value"
+    v-bind="$attrs"
+    @blur="looseFocus"
+    data-auto-focus="true"
+  />
+</template>
+
+<script>
+export default {
+  name: 'AutoFocusInput',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    initialValue: {
+      type: String,
+      default: ''
+    }
+  },
+  computed: {
+    refName() {
+      return `auto-focus-input-${this.name}`
+    }
+  },
+  watch: {
+    value () {
+      this.$emit('input', this.value)
+    }
+  },
+  data () {
+    return {
+      value: this.initialValue,
+      isFocus: false
+    }
+  },
+  methods: {
+    // AutoFocusInput is only activated
+    enable () {
```

Comment:
I don't understand this method: looks like it should be a getter that tells whether the element `isEnabled`, but then it should be named `isFocused`. But then, in this case, I'd just go like

```javascript
return this.refs[this.refName].hasFocus()
```

Reply:
This method is to prevent to focus the AutoFocusInput if the user is focusing another input.  

I cannot use a computed because I need to get the current active element from the DOM each time and computed will cache the result and never compute it again since I don't change the state.  

I renamed it to `shouldAcquireFocus` for more clarity